sdk/schemaregistry/schema-registry/samples/v1-beta/typescript/src/getSchemaByVersion.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Demonstrates the use of getSchemaByVersion.
+ */
+
+import { DefaultAzureCredential } from "@azure/identity";
+import {
+  SchemaRegistryClient,
+  SchemaDescription,
+  KnownSchemaFormats,
+} from "@azure/schema-registry";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+// Set these environment variables or edit the following values
+const fullyQualifiedNamespace =
+  process.env["SCHEMAREGISTRY_AVRO_FULLY_QUALIFIED_NAMESPACE"] || "<fullyQualifiedNamespace>";
+const groupName = process.env["SCHEMA_REGISTRY_GROUP"] || "AzureSdkSampleGroup";
```

Comment:
I was planning on making one of my samples use json or custom instead of avro, just to show that we support multiple formats. Not something that you def need to change, but leaving it here in case you want to :)

Reply:
I see, I don't feel strongly about it. I think the new enum + the portal UI should help customers discover supported formats.

Reply:
sounds good